activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/test/java/org/activiti/cloud/services/modeling/mock/MockFactory.java

```diff
@@ -109,14 +106,20 @@ public static ModelEntity processModelWithExtensions(ProjectEntity parentProject
         ModelEntity processModel = new ModelEntity(name,
                                                    PROCESS);
         processModel.setProject(parentProject);
-        processModel.setExtensions(extensions!=null?extensions.getAsMap():null);
+        processModel.setExtensions(extensions!=null?generateExtensions(name, extensions):null);
         if (content != null) {
             processModel.setContentType(CONTENT_TYPE_XML);
             processModel.setContent(content);
         }
         return processModel;
     }
 
+    private static Map<String, Object> generateExtensions(String name, Extensions extension) {
```

Comment:
why generate? what are trying to do here?

Reply:
fixed